src/pages/iou/request/step/IOURequestStepAmount.tsx

```diff
@@ -56,22 +46,22 @@ function IOURequestStepAmount({
     transaction,
     splitDraftTransaction,
     draftTransaction,
-}) {
+}: IOURequestStepAmountProps) {
     const {translate} = useLocalize();
-    const textInput = useRef(null);
-    const focusTimeoutRef = useRef(null);
+    const textInput = useRef<BaseTextInputRef | null>(null);
+    const focusTimeoutRef = useRef<NodeJS.Timeout | null>(null);
     const isSaveButtonPressed = useRef(false);
-    const originalCurrency = useRef(null);
+    const originalCurrency = useRef<string | null>(null);
     const iouRequestType = getRequestType(transaction);
     const isEditing = action === CONST.IOU.ACTION.EDIT;
     const isSplitBill = iouType === CONST.IOU.TYPE.SPLIT;
     const isEditingSplitBill = isEditing && isSplitBill;
-    const {amount: transactionAmount} = ReportUtils.getTransactionDetails(isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction : transaction);
-    const {currency} = ReportUtils.getTransactionDetails(isEditing ? draftTransaction : transaction);
+    const {amount: transactionAmount} = ReportUtils.getTransactionDetails(isEditingSplitBill && !lodashIsEmpty(splitDraftTransaction) ? splitDraftTransaction : transaction) ?? {amount: 0};
+    const {currency} = ReportUtils.getTransactionDetails(isEditing ? draftTransaction : transaction) ?? {currency: CONST.CURRENCY.USD};
 
     useFocusEffect(
```

Comment:
I think I could use `useAutoFocusInput` here if needed.

Reply:
Let's not refactor it here, unless you think it's a 10 mins fix

Reply:
I noticed this approach being used in multiple places, so I will keep it as is. However, we should consider using the `useAutoFocusInput` hook throughout the entire app to clean things up.